regression-tests.api/test_Zones.py

```diff
@@ -59,13 +62,22 @@ def test_list_zones(self):
         print(example_com)
         required_fields = ['id', 'url', 'name', 'kind']
         if is_auth():
-            required_fields = required_fields + ['masters', 'last_check', 'notified_serial', 'edited_serial', 'serial', 'account']
+            required_fields = required_fields + ['masters', 'last_check', 'notified_serial', 'serial', 'account']
+            if dnssec:
+                required_fields = required_fields = ['dnssec', 'edited_serial']
             self.assertNotEquals(example_com['serial'], 0)
         elif is_recursor():
             required_fields = required_fields + ['recursion_desired', 'servers']
         for field in required_fields:
             self.assertIn(field, example_com)
+        if not dnssec:
```

Comment:
I think this should be inside the `if is_auth()` block

Reply:
Fixed, and now we also don't pointlessly test `dnssec=false` on the Recursor.